package.json

```diff
@@ -30,13 +30,13 @@
   "author": "devsupport@webex.com",
   "license": "MIT",
   "peerDependencies": {
-    "rxjs": "^6.5.4",
-    "webex": "^1.157.0"
+    "rxjs": "^6.6.7",
+    "webex": "^1.159.4"
```

Comment:
We don't want to update rxjs and webex version Harsh. Is there a need in this PR?

Reply:
changes done